### PR TITLE
Add scene audio editor

### PR DESCRIPTION
**Pull request by bigfoodK**
image
- list audio assets
- set bgm
- set volume
- preview on hovering
   - 1~2 sec delayed

### CODE REVIEW

File: luda-editor/new-client/src/audio_util/audio_storage.rs [hunk 1]
```diff
@@ -0,0 +1,84 @@
+use namui::*;
+use std::{
+    collections::HashMap,
+    error::Error,
+    sync::{Arc, RwLock},
+};
+
+lazy_static! {
+    static ref PSD_SPRITE_STORAGE: AudioStorage = AudioStorage::new();
```
Comment (namse) [inline luda-editor/new-client/src/audio_util/audio_storage.rs:9]: Name should be audio storage?

File: luda-editor/new-client/src/episode_editor/scene_audio_editor/audio_select_tool.rs [hunk 2]
```diff
@@ -0,0 +1,227 @@
...
+        let (hovering, set_hovering) = ctx.state::<Option<Hovering>>(|| None);
+        let (play_handle, set_play_handle) = ctx.state(|| None);
+
+        ctx.interval("play audio if hovering", 1.sec(), |_| {
+            let Some((Hovering { started_at }, audio)) =
+                hovering.as_ref().as_ref().zip(audio.as_ref())
+            else {
+                return;
+            };
+            namui::log!("{:?}", started_at);
```
Comment (namse) [inline luda-editor/new-client/src/episode_editor/scene_audio_editor/audio_select_tool.rs:184]:
```suggestion
```

**Review by namse — CHANGES_REQUESTED** [review]

File: luda-editor/new-server/rpc/src/types/scene.rs [hunk 3]
```diff
@@ -1,4 +1,5 @@
 use migration::schema::SceneDoc;
+use migration::schema::SceneSound as SceneSound_;
 
 pub type Scene = SceneDoc;
-
+pub type SceneSound = SceneSound_;
```
Comment (namse) [inline luda-editor/new-server/rpc/src/types/scene.rs:5]: Why not just `pub use`?

**Review by namse — CHANGES_REQUESTED** [review]

File: luda-editor/new-server/rpc/src/types/scene.rs (hunk 3 above)
Comment (bigfoodK) [reply luda-editor/new-server/rpc/src/types/scene.rs:5]: for unity.`pub type Scene = SceneDoc;`-like pattern.

Comment (bigfoodK) [reply luda-editor/new-server/rpc/src/types/scene.rs:5]: I think so

**Review by namse — APPROVED** [review]

**Pull request merged**